pkg/common/helper/helper.go

```diff
@@ -37,6 +37,17 @@ func New() *H {
 	}
 
 	ginkgo.BeforeEach(h.Setup)
+
+	return h
+}
+
+func NewOutsideGinkgo() *H {
+	h := &H{
```

Comment:
Let's extract this logic into a helper function and have New() and NewOutsideGinkgo() both call it. It's trivial, I know, but will reduce the amount of repetition here.